database/migrations/dbConfig.js

```diff
@@ -0,0 +1,5 @@
+const knex = require('knex');
+
+const knexConfig = require('../knexfile.js');
+
+module.exports = knex(knexConfig.development);
```

Comment:
for future reference, we'll typically end up using a dotenv value here, usually called `NODE_ENV`:
```javascript
// or dev. in case a local .env file isn't set up
const env = process.env.NODE_ENV || 'development';

module.exports = knex(knexConfig[env]);
```

Reply:
That way, we'll be able to use different configs (production, testing, etc.) dynamically.